crates/turbopack-ecmascript/src/chunk/item.rs

```diff
@@ -43,26 +38,35 @@ pub struct EcmascriptChunkItemOptions {
 #[turbo_tasks::value_trait]
 pub trait EcmascriptChunkItem: ChunkItem {
     fn content(&self) -> EcmascriptChunkItemContentVc;
-    fn chunking_context(&self) -> ChunkingContextVc;
+    fn chunking_context(&self) -> EcmascriptChunkContextVc;
     fn id(&self) -> ModuleIdVc {
-        EcmascriptChunkContextVc::of(self.chunking_context()).chunk_item_id(*self)
+        self.chunking_context().chunk_item_id(*self)
     }
```

Comment:
We should move that into `impl EcmascriptChunkItemVc {}`. It shouldn't be allowed to change this method from the EcmascriptChunkItem impl